EnglishFilter/sections/general_extensions.txt

```diff
@@ -95,7 +95,7 @@ bitporno.com#%#(function(){var b=window.setTimeout;window.setTimeout=function(a,
xx.com,tarjetarojaonline.org,rojadiracta.me,streamshunter.tv,imageweb.ws,freepornhq.xxx,sextop.net,recwebcam.com,firstrowi.com,streamhunter.top,vipbox.me,vipboxing.eu,roketfilmizle.org,atdhe.ru,justpicsplease.com,kisshentai.net,katyporn.net,xvideos5.com.br,cam-archive.com,atdhe.top,zedporn.com,julesjordanvideo.com,ddl-mdh.org,pornomineiro.com,yeswegays.com,myp1p.eu,3ddl.tv,fullxxxmovies.net,warez-world.org,myp2p.biz,videosdesexoxxx.blog.br,piratecams.com,freepornstreams.org,drago99.com,cricfree.org,mamasijaya.me,vipbox.so,stream2watch.ru,mypornstarbook.net,sexhayvc.com,crickfree.net,atdhe.to,atdhe.cc,vrsmash.com,webdesigndev.com,line25.com,stream2watch.eu,doitruy.net,plashporn.com,tarjetarojaonline.eu,sportlemons.com,javhdmovies.net,redvideos.com.br,rojadirectas.me,xxhay.com,handjobhub.com,wiziwig.to,femdomtube.xxx,porntube4k.net,sexne.net,firstsrowsports.com,darkom.me,1kino.in,firstrow1uk.eu,lavoratorio.it,scatkings.com,losporn.com,wiziwig.ru,sportlemon.net,entrejuegosgratis.com,ultradownloads.com.br,drakulastream.tv,javstreams.org,xnxxhamster.net,downloadableporn.org,lshunters.tv,frontrowsports.ru,wisiwig.eu,xxxvideosporno.blog.br,amateurbusters.com,pornclipsxxx.com,youtubeconvert.cc,pornstreams.org,freshmusic.download,firstrowsportes.tv,ilivestream.com,porn2dl.net,sportlemon.top,sportcategory.org,guyswithiphones.com,streamhunter.net,astu-maison.com,deep-warez.org,xxx-photo.com,pornogiusto.com,vipbox.biz,frontrowsport.tv,ilemi.so,streamhunter.so,atdhe.se,camarchiver.com,phimmup.net,realstreamunited.tv,jphimsex.net,kissdoujin.com,astucito.com,exposedlyrics.com,torrentz.in,sexsub.net,astu-cuisine.com,pussylicking.me,pornotube.it,xxxsiterips.xyz,videojav.net,lyriczz.com,sexiestpicture.com,fullpornmovies.net,pornbust.net,krank.de,hentaidream.org,freestream.to,firstrow.top,myp2p.sx,creamypussyvideos.com,firstrows.net,pirateporn.net,freepornpass.biz,pelis24hrs.com,hahasports.net,cphimsex.com,sportlemons.net,nastyplace.org,p2p4u.tv,sessoitaliano.com,ddload.tv,xxxphoto.org,somosgnula.com,lawanswers.com.au,couplecams.video,spintissimo.com,blow-jobs.me,firstsrowsports.eu,ilemi.co,newxxx.xyz,wiziwigs.eu,incesti.com,sportlemons.org,xfiles.cc,goatd.me,freestreams.eu,celebritysextapes.org,wiziwig.top,pornotubo.it,divxtotal.com,verseries.pw,firstsrow.eu,streams2watch.me,pornilly.com,firstrowi.eu,yougoggle.com,firstrowi.net,pornincest.org,buyu0910.com,pornsextube.tv,vive-le-porno.com,firstrow1us.eu,hahasport.top,gigaporno.com,webcamrip.com,hahasports.tv,streamsport.eu,adultxpedia.com,escorte.sex,xhamster.it,velhas10.com,bibledusexe.com,atdhe.me,bigboobedporn.xyz,annangelxxx.com,globaltravel.com,stopstreams.tv,coolsport.biz,theassfactory.com,atdhe.mx,olpair.com,streamplay.gdn,filecrypt.cc,theteenbay.co,turbobit.net,sankakucomplex.com,pervclips.com,fuckhardporn.com,streamplay.to,scnlog.eu,streamango.com,solidfiles.com,uploaded.net,safelinking.net,openload.co#%#Object.defineProperty(window, 'BetterJsPop', { get: function() { return { init: function() { } }; } });
```

Comment:
Don't work on my end. Btw, let's give a chance to another pull request https://github.com/AdguardTeam/AdguardFilters/pull/74961